tap_google_analytics/client.py

```diff
@@ -398,11 +403,16 @@ def schema(self) -> dict:
         #  {start_date, end_date} params as keys
         if not date_dimension_included:
             self.logger.warn(
-                f"Incrmental sync not supported for stream {self.tap_stream_id}, \
+                f"Incremental sync not supported for stream {self.tap_stream_id}, \
                     'ga.date' is the only supported replication key at this time."
             )
             primary_keys.append("report_start_date")
             primary_keys.append("report_end_date")
 
+        if self.include_view_id_in_output:
```

Comment:
```suggestion
        if self.config.get("include_view_id_in_output", False):
```

@menzenski I like your way of defining this in the init a shared but it looks like the super() in the init uses this before `include_view_id_in_output` is set so it cant find `include_view_id_in_output`. I'm open to other alternatives but heres a suggestion that worked for me.